ipatests/prci_definitions/prci_checker.py

```diff
@@ -0,0 +1,181 @@
+#! /usr/bin/python3
+
+import os
+import yaml
+import glob
+import click
```

Comment:
As this tool was intented to be just a manual tool for devs, I just thougth of it as nice feature for better UX. But of course, if we think about integrating it into some automated pipeline, I could delete them to avoid any trouble.

Also from [Click docs](https://click.palletsprojects.com/en/7.x/utils/#ansi-colors):

> Click’s [echo()](https://click.palletsprojects.com/en/7.x/api/#click.echo) function will automatically strip ANSI color codes if the stream is not connected to a terminal.

Reply:
It was my first choice since it was convenient and I saw we use it in other scripts like https://github.com/freeipa/freeipa-perftest/blob/master/src/ipaperftest/core/main.py
But I wouldn't have a problem switching to argparse, as I don't use advanced feature (besides styles).

Reply:
I agree with @stanislavlevin. There is no real reason to use `click` here. Please stick to what the project already uses. As of note, `freeipa-perftest` is a separate project so its decisions aren't applying to `freeipa` project. ;)